variables.tf

```diff
@@ -55,10 +212,16 @@ variable "notifications" {
   }
 }
 
+variable "create_policy" {
```

Comment:
As mentioned on the resource, I think this variable can be removed

Reply:
Removed